addons/captives/XEH_postInit.sqf

```diff
@@ -1,4 +1,5 @@
 #include "script_component.hpp"
+#include “\a3\editor_f\Data\Scripts\dikCodes.h”
```

Comment:
Those are some wrong quotes.

Reply:
Whoops, this happens when directly copying from the cba docu